MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -68,8 +73,9 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
 Note: Frontend criteria apply to both modules and shared libraries. 
 
 * If provided, End-to-end tests must be written in an [officially supported technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1] (3, 4)
-  * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
+  * -_note: If the new module works together with existing modules, the team must provide a result of an integration test.
   * -_note: these tests are defined in https://github.com/folio-org/stripes-testing_
+* Front-end unit tests are written in Jest/RTL and reach at least 80% coverage
```

Comment:
This overlaps / is more specific than the Shared/Common criterion:
> * Unit tests have 80% coverage or greater, and are based on [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1] (3, 4)

We should probably either define unit test requirements in both FE & BE sections, or in the Common section, but not both.

Reply:
I agree with this, I find it would be confusing otherwise.

Reply:
Agreed and changed. I missed that FE Unit tests are already described in the Shared/Commons section + the link therein.